internal/memorylimiter/memorylimiter.go

```diff
@@ -168,16 +159,6 @@ func getMemUsageChecker(cfg *Config, logger *zap.Logger) (*memUsageChecker, erro
 func (ml *MemoryLimiter) readMemStats() *runtime.MemStats {
 	ms := &runtime.MemStats{}
 	ml.readMemStatsFn(ms)
-	// If proper configured ms.Alloc should be at least ml.ballastSize but since
-	// a misconfiguration is possible check for that here.
-	if ms.Alloc >= ml.ballastSize {
```

Comment:
Can we add a separate changelog entry for this change?

Reply:
added one for each memory limiter